src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,41 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter DATE_TIME_FORMATTER =
+            DateTimeFormatter.ofPattern("dd.MM.yyyy");
+    private static final int DATE_INDEX = 0;
+    private static final int NAME_INDEX = 1;
+    private static final int HOURS_OF_WORK_INDEX = 2;
+    private static final int INCOME_PER_HOUR_INDEX = 3;
+    private static final String SEPARATOR = System.lineSeparator();
```

Comment:
it's a redundant constant, remove it

Reply:
But why? Is it better to write eg. "if (splittedLine[1].equals(name)" instead of "if (splittedLine[NAME_INDEX].equals(name)"?
Code is more readable with [NAME_INDEX] (I suppose).